migrations/915-rename-addon-tags.sql

```diff
@@ -0,0 +1 @@
+RENAME TABLE `users_tags_addons` TO `addons_tags`;
```

Comment:
I was trying to find docs on whether this is a simple rename vs copying data to a new table and found that all FK references also need to be updated. Ugh.